### PR TITLE
Add player object

### LINKED ISSUE #3: Create character that can move

### PR DESCRIPTION
**Pull request by LvanBart**
- Added a player prefab, with movement
- Added an instance of player to the beta level

Closes #3

### CODE REVIEW

File: HobgobDie/Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    private bool isGrounded = false;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb = this.gameObject.GetComponent<Rigidbody2D>();
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (TroyHouston) [inline HobgobDie/Assets/Scripts/PlayerMovement.cs:20]: This should probably be inside `FixedUpdate()`
https://stackoverflow.com/questions/34447682/what-is-the-difference-between-update-fixedupdate-in-unity

Comment (LvanBart) [reply HobgobDie/Assets/Scripts/PlayerMovement.cs:20]: I initially tried to put the keydown stuff in fixed update but it didn't register every keypress. I moved it to update as per https://answers.unity.com/questions/294079/unity-keydown-doesnt-always-register.html, and it worked.

Comment (TroyHouston) [reply HobgobDie/Assets/Scripts/PlayerMovement.cs:20]: I see!
I think these two links show the ideal approaches to this:
- https://docs.unity3d.com/ScriptReference/Rigidbody-velocity.html
- https://answers.unity.com/questions/1534204/input-in-update-and-physics-in-fixed-update-good-p.html

File: HobgobDie/Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public float movementScalar;
+    public float maxSpeed;
+    public float jumpScalar;
+
+    private Rigidbody2D rb;
```
Comment (TroyHouston) [inline HobgobDie/Assets/Scripts/PlayerMovement.cs]: Fullname! 🐱‍🚀

**Review by TroyHouston — APPROVED** [review]
Looks great!

File: HobgobDie/Assets/Scripts/PlayerMovement.cs (hunk 2 above)
Comment (LvanBart) [reply HobgobDie/Assets/Scripts/PlayerMovement.cs]: Renamed variable.

File: HobgobDie/Assets/Scripts/PlayerMovement.cs (hunk 1 above)
Comment (LvanBart) [reply HobgobDie/Assets/Scripts/PlayerMovement.cs:20]: Thanks, that worked!

**Pull request merged**